DOC Update zero-code-change documentation (#6697)
rapidsai/cuml: docs/source/zero-code-change.rst

.. code-block::

   %load_ext cudf.pandas

>    from cuml.accel import install

Reviewer: I believe

```
%load_ext cudf.pandas
%load_ext cuml.accel
```

_should_ work, and would be cleaner than telling the user to call `install` directly themselves.

If that doesn't work, it's probably due to double-modifying `rmm` settings, we could probably detect in `cuml` if rmm has been changed from the defaults and skip setting up uvm in that case to make that work.

Reviewer: I can confirm that running these in either order works today (though doing cuml before cudf raises an innocuous warning log from cudf saying a managed memory resource is already configured.

I'm going to make some changes to how we handle managed memory and add a test for this, just noting that combining the magics works already today.

Reviewer: https://github.com/rapidsai/cuml/pull/6700

   install()

> A single magic invocation is planned for a future release of cuML.

Reviewer: Do we want to say something even that strong? I'm loathe to promise things in docs - personally I'd drop this entirely, or maybe hedge more like

```suggestion
A single magic invocation may be possible in a future release of cuML.
```

or maybe say "future release of RAPIDS" since we might want to unify the magics used here.

Reviewer 2: I'd remove the whole sentence about "single magic" and what might be in the furture. It isn't actionable for the user.

Using two `%load_ext` lines is super good enough and simple to understand. Users who learnt how to use cudf.pandas don't have to relearn how to use it when they want to start using cul.accel (and vice versa).

The important thing is that they actually compose and enabling both at the same time (in either order) doesn't break things.

Author: I'll fully remove that.

Author: Done in fab415e0731c3cce203757e47a0c3eda992246f1 .
